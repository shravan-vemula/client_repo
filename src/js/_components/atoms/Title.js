import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { myTheme } from "theme";
import BtrackerLogo from "images/Btracker.png";

const useStyles = makeStyles((theme) => ({
  title: {
    flexGrow: 1,
  },
  base: {
    marginLeft: "100px",
    paddingTop: "5px",
    width: "300px",
    height: "38px",
    fontSize: "32px",
    fontWeight: "500",
    fontStretch: "normal",
    fontStyle: "normal",
    lineHeight: "normal",
    letterSpacing: "normal",
    fontFamily: myTheme.typography.fontFamily.default,
  },
}));

const Title = () => {
  const classes = useStyles();
  return (
    <div
      className={classes.base}
      className={classes.title}
      style={{ paddingLeft: "80px" }}
    >
      <div id="logo" className="logoBtracker">
        <img src={BtrackerLogo} className="BtrackerIcon" />
      </div>
    </div>
  );
};
export default Title;
