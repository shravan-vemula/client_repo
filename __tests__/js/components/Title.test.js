import React from "react";
import { cleanup } from "@testing-library/react";
import Title from "../../../src/js/_components/atoms/Title";

afterEach(cleanup);

describe("Title Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
});
