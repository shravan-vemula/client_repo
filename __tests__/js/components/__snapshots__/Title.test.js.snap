// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component renders correctly 1`] = `ShallowWrapper {}`;
